master/buildbot/test/unit/test_www_oauth.py

```diff
@@ -161,6 +161,11 @@ def test_GithubVerifyCode(self):
                           'username': 'bar',
                           'groups': ["hello", "grp"],
                           'full_name': 'foo bar'}, res)
+        res = yield self.githubAuth.acceptToken("TOK3N")
```

Comment:
I am seeing GeneratorExit errors in the Python 3 build log:
https://nine.buildbot.net/#/builders/2/builds/109615/steps/5/logs/stdio

They are not causing the build to fail, but they need to be investigated at some point.